plugins/scaffolder-common/src/permissions.ts

```diff
@@ -156,5 +157,4 @@ export const scaffolderPermissions = [
   ...scaffolderTemplatePermissions,
   ...scaffolderActionPermissions,
   ...scaffolderTaskPermissions,
-  templateManagementPermission,
```

Comment:
mmm this used to be correct 🤔 
I think the problem should be somewhere else then

Reply:
So I think the way to go would be to pass `permissions: scaffolderPermissions` here: https://github.com/backstage/backstage/blob/c34fd4ed4ab9ac2e90d51c988d259b5e4be4f0b6/plugins/scaffolder-backend/src/service/router.ts#L448
but I'm not sure whether the integration router can properly handle duplicates at this point, as `scaffolderPermissions` contains resources permissions as well.
Short term another way would be to pass `permissions: [...scaffolderPermissions, templateManagementPermission]`

Reply:
I'll give this a try, with the current fix there was no duplicates based on my testing.

Reply:
your fix should work. The issue is to group the new permission under `scaffolderTemplatePermissions` as they are resource permissions of type `scaffolder-template`